ocs_ci/ocs/ui/deployment_ui.py

```diff
@@ -292,7 +284,8 @@ def install_ocs_ui(self):
         Install OCS via UI
 
         """
-        self.add_disks_lso()
+        if config.DEPLOYMENT.get("local_storage"):
+            add_disk_for_vsphere_platform()
```

Comment:
I would add another condition for checking if the platform is vsphere. If not, skip the test